plugins/modules/digital_ocean.py

```diff
@@ -433,7 +448,7 @@ def main():
             size_id=dict(),
             image_id=dict(),
             region_id=dict(),
-            ssh_key_ids=dict(type='list', no_log=False),
+            ssh_key_ids=dict(type='list', elements='str', no_log=False),
```

Comment:
This change potentially should have a changelog fragment ("``ssh_key_ids`` list entries are now validated to be strings").